### PR TITLE
header fixed and made responsive

### PR DESCRIPTION
**Pull request by MussadiqKhan**

### CODE REVIEW

**Comment by MussadiqKhan** [comment]
Homepage body responsive

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
+        <label htmlFor="menu-btn" className="menu-icon" id="menu">
+          <input style={{ display: "none" }} />
+          <span className="navicon"></span>
+        </label>
+        <ul className="menu">
+          <li>
+            <Link to="/about">Home</Link>
+          </li>
+          <li>
+            <Link to="/what-we-do">About</Link>
```
Comment (mgreiler) [inline src/components/Navbar.js:28]:
```suggestion
            <Link to="/about">About</Link>
```

File: src/components/Navbar.js [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
+          id="menu-btn"
+          //onClick={onClick}
+        />
+        <label htmlFor="menu-btn" className="menu-icon" id="menu">
+          <input style={{ display: "none" }} />
+          <span className="navicon"></span>
+        </label>
+        <ul className="menu">
+          <li>
+            <Link to="/about">Home</Link>
```
Comment (mgreiler) [inline src/components/Navbar.js:25]:
```suggestion
            <Link to="/">Home</Link>
```

File: src/components/Navbar.js [hunk 3]
```diff
@@ -1,19 +1,37 @@
...
+          </li>
+          <li>
+            <Link to="/what-we-do">About</Link>
+          </li>
+          <li>
+            <Link to="/how-we-do-it">Sponsor Us</Link>
```
Comment (mgreiler) [inline src/components/Navbar.js:31]:
```suggestion
            <Link to="/sponsorship">Sponsor Us</Link>
```

**Review by mgreiler — COMMENTED** [review]
I've changed the link destination to the actual pages.

**Pull request merged**